corehq/apps/zapier/tests.py

```diff
@@ -58,7 +65,7 @@
 """
 
 FORM_XMLNS = "https://www.commcarehq.org/test/zapier/"
-CASE_TYPE = "https://www.commcarehq.org/test_case/zapier/"
+CASE_TYPE = "lemon meringue pie"
```

Comment:
I think `lemon-meringue-pie` might be better. You can't have spaces in a case type name.